### PR TITLE
XLA - Batch Norm - Bessel's correction generates NaNs in FP16

### PR DESCRIPTION
**Pull request by georgepaw**
For FP16 this means that the factor will be a NaN for sufficiently big inputs.

### CODE REVIEW

**Comment by jlebar** [comment]
@rthadur Please see https://github.com/tensorflow/tensorflow/pull/30219#issuecomment-506764404.

**Review by sanjoy — APPROVED** [review]

**Comment by rthadur** [comment]
> @rthadur Please see [#30219 (comment)](https://github.com/tensorflow/tensorflow/pull/30219#issuecomment-506764404).

Thanks for letting me know.
